examples/restful-ws-quarkus/src/test/java/io/cloudevents/examples/quarkus/UserResourceTest.java

```diff
@@ -3,8 +3,8 @@
 import io.quarkus.test.junit.QuarkusTest;
 import org.junit.jupiter.api.Test;
 
+
```

Comment:
What's the reason for this empty line? Can we remove it?

Reply:
@pierDipi  I will remove it and resbumit this pr later.

Reply:
```suggestion
```

You can alternatively accept this suggestion